docs/zh/start/trace/zipkin.md

```diff
@@ -0,0 +1,93 @@
+# ZipKin trace 接入
+
+## 配置
+
+示例：configs/config_trace_zipkin.json
+
+```json
+{
+  "tracing": {
+    "enable": true,
+    "driver": "Zipkin",
+    "config": {
+      "config": {
+        "config": {
+          "service_name": "layotto",
+          "reporter_endpoint": "http://127.0.0.1:9411/api/v2/spans",
+          "recorder_host_post": "127.0.0.1:9000"
+        }
+      }
+    }
+  }
+}
+
+```
+| 字段   | 必填  | 说明                                           |
+|------|-----|----------------------------------------------|
+| service_name | Y   | 服务名称，默认为layotto                              |
+| reporter_endpoint | Y   | 链路日志输出，默认为http://127.0.0.1:9411/api/v2/spans |
+| recorder_host_post     | Y   | 当前服务端口号，默认为127.0.0.1:9000                                  |
```

Comment:
What's the difference between `recorder_host_post` and `reporter_endpoint`? Could u add some explanation for them?